docs/guides/examples/guide-syslog-config.yaml

```diff
@@ -0,0 +1,19 @@
+syslog:

```

Comment:
Since there are multiple parameters which are configurable for syslog, I felt having all these parameters grouped under a config file would be user-friendly. Please let me know your view on this, I will make the changes accordingly!

Reply:
Unfrotunately, using a config file here will be inconsistent with other configuration parameters that components have. Adding file configuraiton will mean that some configuration will be defined as a command-line flag and logging configuration will be file-based which seems confusing.

In order to simplify the configuration I would suggest to use a DNS format for syslog target with the following format: `protocol://user:password@host:port`
This allows to use a single flag to configure 5 options at once. The rest of configuration parameters will need to be flag-based.